crates/host/src/wasmbus/mod.rs

```diff
@@ -4264,78 +4260,86 @@ impl Host {
         }
     }
 
-    /// Publishes `count` `actor_started` events and one `actors_started` event with the supplied count
-    async fn publish_actor_started_events(
+    // TODO(#1092): only publish actor_scale_failed event after wasmCloud releases 0.81
```

Comment:
Now that 0.81 is out, can we resolve these TODOs?

Reply:
I actually have to change these to 0.82 😢 The idea was to get these events published but not remove the old ones until a minor bump is done